src/components/Header/Header.tsx

```diff
@@ -85,7 +85,7 @@ export default function Header() {
             <SearchButton isBlack={isBlack} />
             {!users ? <LoginButton onClick={handleLoginButtonClick} /> : <LogoutButton />}
             <button className={cn('user-icon')} type='button' onClick={handleUserIconClick}>
-              {profileImage ? (
+              {profileImage && profileImage !== 'null' ? (
```

Comment:
```suggestion
              {profileImage && profileImage != null ? (
```
요즘 자바스크립트 공부하면서 드는 생각인데 이런식으로 null이 문자열로 넘어오는 경우에는 동치연산자 말고 동등 연산자로 사용해도 좋을거같아요
물론 컴포넌트 수정할때 제가 고려하지 못한 부분이지만.. 암튼

Reply:
해봤는데 인식을 못해요ㅜㅜ